src/app/shared/card/card-treatments/card-treatments.component.html

```diff
@@ -0,0 +1,19 @@
+<div *ngIf="data" class="card-treatment">
+  <div class="title-container">
+    <div class="title">{{ drugInfo[0] }}</div>
+  </div>
+  <app-date-card
+    *ngIf="data.treatmentDateTime"
+    [date]="data.treatmentDateTime"
+    [label]="time"
+    [showYear]="false"
+    class="date"
+  ></app-date-card>
+  <div class="separator"></div>
+  <div class="drug">{{ drugInfo[1] }}</div>
```

Comment:
drugInfo[0], drugInfo[1] same thing

Reply:
drugInfo cannot be an empty array, as some of the field values which are concatenated in it are required fields. Anyway, tested it with [] and no error appeared.